tests/extensions/extra/tables.html

```diff
@@ -38,22 +38,22 @@ <h2>Table Tests</h2>
 <table>
 <thead>
 <tr>
-<th align="left">Item</th>
-<th align="right">Value</th>
+<th style="text-align: left;">Item</th>
+<th style="text-align: right;">Value</th>
```

Comment:
Any tests which have been replicated in `tests/test_syntax/extensions/test_tables.py` should be removed from this file. If that leaves this file empty, then it can be deleted long with the entry for it in [tests/test_legacy.py](https://github.com/Python-Markdown/markdown/blob/master/tests/test_legacy.py#L185)